webapp/channels/src/components/autosize_textarea.tsx

```diff
@@ -156,14 +156,17 @@ export class AutosizeTextarea extends React.PureComponent<Props> {
         return (
             <div>
                 {textareaPlaceholder}
+                <label
+                    id='placeholderAriaLabel'
+                    style={{display: 'none', visibility: 'hidden', height: 0}}
```

Comment:
Same comments here regarding the ID needing to be unique and that this should use the `sr-only` class. This component already has a `this.props.id`, so we can make the label's ID `id={id + '-label'}` to match the other IDs in this file